project_national/housing_characteristics/Neighbors.tsv

```diff
@@ -4,3 +4,5 @@ Single-Family Detached	1.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.0
 Single-Family Attached	0.000000	0.005808	0.104524	0.154273	0.286172	0.410100	0.039122
 Multi-Family with 2 - 4 Units	0.000000	0.005808	0.104524	0.154273	0.286172	0.410100	0.039122
 Multi-Family with 5+ Units	0.000000	0.005808	0.104524	0.154273	0.286172	0.410100	0.039122
+# Source: Microsoft Footprint Database (Accessed 2018) for Multi-Family and Single-Family Attached							
```

Comment:
I mistakenly told you that we used "Microsoft Footprints Database". We have not used that yet. It should read "OpenStreetMap data queried by Radiant Labs"